tests/robottelo/test_hammer.py

```diff
@@ -109,189 +109,189 @@ def test_parsed_json_match_parsed_csv(self):
         assert hammer.parse_json(json_output) == hammer.parse_csv(csv_ouput_lines)[0]
 
 
-class TestParseHelp:
-    """Tests for parsing hammer help output"""
+# class TestParseHelp:
```

Comment:
@JacobCallahan Fixing this test here in https://github.com/SatelliteQE/robottelo/pull/10375, so this failing test can be uncommented now

Reply:
@Gauravtalreja1 updated, thanks for addressing that.